debugger/src/cell_renderers/cellrendererbreakicon.c

```diff
@@ -284,7 +328,11 @@ static void cell_renderer_break_icon_init (CellRendererBreakIcon *cell)
 	cell->condition = NULL;
 	cell->hitscount = 0;
 	
+#if GTK_CHECK_VERSION(3, 0, 0)
+	g_object_set(cell_renderer, "mode", GTK_CELL_RENDERER_MODE_ACTIVATABLE, NULL);
+#else
 	cell_renderer->mode = GTK_CELL_RENDERER_MODE_ACTIVATABLE;
+#endif
```

Comment:
Here the GTK3 version would work on GTK2 too

Reply:
Fixed.